source/opt/ir_builder.h

```diff
@@ -27,32 +27,30 @@ namespace opt {
 constexpr uint32_t kInvalidId = std::numeric_limits<uint32_t>::max();
```

Comment:
I think this constexpr will blow up on VS2013 as well.
You can use  either
   const uint32_t kInvalidId = ....
or
  enum : uint32_t {  kInvalidId = ... } ;